src/App.tsx

```diff
@@ -1,39 +1,58 @@
 import React from 'react';
+import { Clock } from './components/Clock';
+
 import './App.scss';
 
+type State = {
+  visible: boolean;
```

Comment:
will be better to name it `isVisible`

Reply:
and specify some noun, what should be visible

Reply:
fixed

> will be better to name it `isVisible`

fixed